avplayback.js

```diff
@@ -193,7 +218,7 @@ instance.prototype.action = function(action) {
 
 			debug('sending ',cmd,"to",self.udp.host);
 
-			self.udp.send(cmd);
+			self.udp.send(cmd + " ");
```

Comment:
If this is not mentioned in the docs, I really would like you to try to find out why this makes it works. Do you have a link to protocol description, I could take a look too.

Reply:
https://www.ifelseware.com/wp-content/uploads/2018/09/AVP_V2_Help/avp_protocol_commands_.htm?ms=AA%3D%3D&st=MA%3D%3D&sct=MA%3D%3D&mw=MjQw

`echo 'AVP|1|LoadClip, 01 ' | nc -w1 -u 192.168.1.1 7000`
`echo 'AVP|1|LoadClip, 01' | nc -w1 -u 192.168.1.1 7000`

Reply:
Could you try with \n instead of space? Remember -e to echo to expand the \n

Reply:
Yes in our testing via echo/nc we tried it with -n as well as through the code + "\n".

Reply:
Did you try with \r\m or \r  too?

I guess we will leave it with a space then, if it works perfectly that way. But could you fix the other issues i commented in your repo, and then I will pull your pull request :)